docs/configure-rest.md

```diff
@@ -185,6 +185,78 @@ exporters, the following parameters can be defined:
 * `instance_labels` (list): display names of labels to export as a separate data-point
 * `include_all_labels` (bool): export all labels with each data-point (overrides previous two parameters)
 
+#### Endpoints
+
+In Harvest REST templates, `endpoints` are additional queries that enhance the data collected from the main query. The main query, identified by the `query` parameter, is the primary REST API for data collection. For example, the main query for a `disk` object is `api/storage/disks`.
+
+Within the `endpoints` section of a Harvest REST template, you can define multiple endpoint entries. Each entry supports its own `query` and associated `counters`, allowing you to collect additional metrics or labels from various API. 
+These `endpoints` collect additional metrics or labels from various API, which are then integrated with the main dataset using a key. This key is denoted by the `^^` notation in the counters of both the main query and the `endpoints`.
```

Comment:
These additional metrics or labels are associated with the main dataset via a key. The key is denoted by the `^^` notation in the counters of both the main query and the `endpoints`.